libs/consumption_metrics/src/lib.rs

```diff
@@ -41,15 +42,25 @@ impl EventType {
     pub fn is_incremental(&self) -> bool {
         matches!(self, EventType::Incremental { .. })
     }
+
+    /// Returns the absolute time, or for incremental ranges, the stop time.
```

Comment:
This is quite the msdn type method comment, but I'd like to communicate with this that either the absolute "at" or incremental "stop" are when (pageserver) records the events.